Gemfile.lock

```diff
@@ -770,4 +770,4 @@ RUBY VERSION
    ruby 2.3.5p376
 
 BUNDLED WITH
-   1.15.4
+   1.16.0.pre.3
```

Comment:
We probably want to stay on the latest stable version of bundler instead of going up to the pre-release version.